porcupine/plugins/statusbar.py

```diff
@@ -99,11 +99,19 @@ def __init__(self, tab: tabs.FileTab):
         self.path_label = ttk.Label(self._top_frame)
         self.path_label.pack(side="left")
         self._line_ending_button = ttk.Button(
-            self._top_frame, command=self._choose_line_ending, style="Statusbar.TButton", width=6
+            self._top_frame,
+            command=self._choose_line_ending,
+            style="Statusbar.TButton",
+            width=6,
+            padding=0,
```

Comment:
The automatically-changing width could be useful for this button too. It doesn't have to be 6 units wide if it only displays a two-character `LF`. In fact, currently a button displaying `LF` is wider than a button displaying `utf-8`:

![Screenshot at 2022-07-25 23-39-21](https://user-images.githubusercontent.com/18505570/180869900-f0d63f00-ecb9-4a8b-9408-291b38bef5eb.png)

If we decide to apply the same padding trick to all buttons, it might be good to add it to the Tcl call in `update_button_style()`